new-pipeline/src/evo.js

```diff
@@ -25,10 +24,14 @@ var gDefaultAggregates = [
   ["95th-percentile", "95th percentile", function (evolution) {
     return evolution.percentiles(95);
   }],
+];
+// these will not appear in the multiselect
```

Comment:
I'm not sure if anyone uses it, but being able to have the submissions on the same plot instead of just on the linked plot below might support use cases requiring a more closer scrutiny of submissions vs values. For now I recommend leaving it in the multiselect.